README.md

```diff
@@ -30,6 +30,14 @@ You can use the `--exclude-region` flag to exclude resources in certain regions
 aws-nuke --exclude-region ap-south-1 --exclude-region ap-south-2
 ```
 
+### Excluding Resources by Age
+
+You can use the `--exclude-since` flag to exclude resources created after a certain time, any resources older than that time will be deleted. For example the following command does not nuke resources created after the midnight on 1st Jan, 2017:
+
+```shell
+aws-nuke --exclude-since '01-01-2017 12:00AM'
```

Comment:
Perhaps `--exclude-after` would be better? Perhaps in the future, it might make sense to add an `--exclude-before` as well.

Reply:
`--exclude-after` sounds better, I had a hard time looking for a better name